falcon/request.py

```diff
@@ -75,15 +75,17 @@ class Request(object):
     Attributes:
         env (dict): Reference to the WSGI environ ``dict`` passed in from the
             server. (See also PEP-3333.)
-        context (dict): Dictionary to hold any data about the request which is
-            specific to your app (e.g. session object). Falcon itself will
-            not interact with this attribute after it has been initialized.
+        context (object): Empty object to hold any data (in its attributes)
```

Comment:
I think it would be helpful to inline an example here to illustrate using attributes in lieu of the (dict) mapping interface.

Reply:
Sure, will address. Just wondering if you meant just exemplifying the "new" way, or also contrasting to the dict interface as in FAQ and changelogs?

Reply:
I think we could add a `Note:` section that says "New in 2.0" or something, explaining that this changed and going forward people should use the attribute-based interface. The regular example would just show the "new" way.